tests/spread/general/passthrough/expected_snap.yaml

```diff
@@ -0,0 +1,23 @@
+name: passthrough-tests
+version: '1.0'
+summary: passthrough-tests
+description: passthrough-tests
+apps:
+  passthrough-none:
+    command: test-cmd
+    command-chain:
+    - snap/command-chain/snapcraft-runner
+    passthrough:
```

Comment:
did you miss this one?

Reply:
fail  :man_facepalming:

Reply:
OK well I don't feel too bad, I just copied the wrong thing at the last minute without looking too hard... whoops!

Cleaned up the test a bit too while I'm at it.